.github/workflows/autofix-command.yml

```
@@ -72,14 +72,17 @@ jobs:
 
             [1]: ${{ steps.vars.outputs.run-url }}
 
+      - name: Get tool versions
+        uses: marocchino/tool-versions-action@v1
+        id: tool-versions
       - name: Set up Poetry
         uses: Gr1N/setup-poetry@v9
         with:
-          poetry-version: "2.0.1"
+          poetry-version: ${{ steps.tool-versions.outputs.poetry }}
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: ${{ steps.tool-versions.outputs.python }}
```

Matrix version should have priority over the tools version file.

```suggestion
          python-version: ${{ matrix.python-version }}
```

(I will apply this change.)